stories/assets/js/tabs.js

```diff
@@ -3,6 +3,4 @@
  */
 export function tabs() {
   $(document).foundation();
```

Comment:
Harish Kumar Kindly use var elem = new Foundation.Tabs(element, options); to init the tabs as it lets us control them dynamically. Also don’t call foundation() on the whole document.